content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -63,7 +63,7 @@ Containers cannot assume an identity from the host because the Security Account
 
 #### Memory reservations {#resource-management-memory}
 
-Windows does not have an out-of-memory process killer as Linux does. Windows always
```

Comment:
The original wording is idiomatic English @budhirajamadhav 

In my view, no change is needed.

Reply:
Ok, no issues. But, in my view- The clear, the better.